### DISCUSSION TITLE
Custom GPT with diagrams

### DISCUSSION THREAD

**Discussion by amolgupta** [mingrammer/diagrams#961] [Show and tell]
Hey folks!
I am working on a custom GPT with chatGPT that supports rendering various types of diagrams for software architecture. I have recently added support for this package and would love to get some feedback. 

Do try it out >> https://chat.openai.com/g/g-D61xRXJME

One of the **known issues** is that ChatGPT would write code with imports like `from diagrams.aws.database import DynamoDB` and `from diagrams.gcp.security import IAM` which my system would not be able to recognize. I tried looking for older versions to see if it's using a specific library version that used these names, but no luck. Any pointers to fix that would be appreciated

**Comment by maichomper** [comment]
@amolgupta this is amazing, I was playing with it the other day and I love the results! Are you keeping this updated? Do you need any help? I would like for my team to adopt it!

**Comment by amolgupta** [comment]
Hi @maichomper , I am glad it is helpful to you! I haven't updated it it last couple of months, but I do get about 50-100 users and around 400 diagrams generated with it everyday consistently, to confidently say that it's still working. ChatGPT was not able to use the correct imports when I wrote it first, so I would fallback to plantUML in case of error, can try to update the prompts again to see if things have changed. Any feedback and feature request is highly appreciated

**Comment by mingrammer** [comment]
This is amazing integration :) Looks cool.
